### PR TITLE
[RFR] Exposed an a11yToggle function on global object

### PR DESCRIPTION
**Pull request by KittyGiraudel**
This PR makes it possible to use dynamically injected toggles.

### CODE REVIEW

**Review by smartmike — APPROVED** [review]
🎉

**Comment by smartmike** [comment]
Nice.

File: example/index.html [hunk 1]
```diff
@@ -282,6 +283,20 @@ <h2 class="title">Connected toggles</h2>
         </div>
       </section>
 
+      <section id="dynamically-injected-toggles">
+        <div class="container">
+          <h2 class="title">Dynamically injected toggles</h2>
+
+          <p>a11y-toggle fires once when the DOM is fully loaded (<code>DOMContentLoaded</code>). You will have to relaunch it if you dynamically add new toggles with JavaScript.</p>
+
+          <p>Thankfully, a11y-toggle exposes an <code>a11yToggle</code> function on the global object.</p>
```
Comment (smartmike) [inline example/index.html:292]: Add something here about the optional context?

Comment (KittyGiraudel) [reply example/index.html:292]: Done.

**Comment by KittyGiraudel** [comment]
YOLO.

**Pull request merged**